extraConfigIPU.py

```diff
@@ -31,13 +35,53 @@ def is_http_url(url: str) -> bool:
         return False
 
 
-def _redfish_boot_ipu(cc: ClustersConfig, imcs: List[str], iso: str) -> None:
+def connect_with_retry(h: host.Host, username: str, password: str) -> None:
+    retries = 10
+    while retries > 0:
+        try:
+            time.sleep(10)
+            retries -= 1
+            h.ssh_connect(username=username, password=password)
+            return
+        except ssh_exception.NoValidConnectionsError:
+            logger.debug("Failed to connect to IMC, retrying")
+
+# By default the ACC does not have network access on boot, we need a workaround to determine the system is up
```

Comment:
how come ACC doesn't have network access on boot? It should call out with dhcp. That's the normal way servers boot. Is the issue that you don't know the IP?

Reply:
The ACC doesn't have a connection to the network following a fresh installation. There is no connection to the internal network initially

Reply:
Think about the setup in a real env. There, we connect the acc to a dhcp server. We should do the sam here.